WindowBoard.cpp

```diff
@@ -18,6 +18,11 @@
 #include <String.h>
 #include <StringView.h>
 
+float prevWidth;
+float prevHeight;
+float defaultWidth;
+float defaultHeight;
```

Comment:
It is bad to have global variables like this, because if any other file does the same with the same names, the variables will conflict. Could you put these inside the GameWindow class instead?

Reply:
Ok.